framework/log/Target.php

```diff
@@ -342,4 +348,17 @@ public function getEnabled()
 
         return $this->_enabled;
     }
+
+    /**
+     * Returns formatted ('Y-m-d H:i:s') timestamp for message.
+     * If [[microtime]] is configured to true it will return format 'Y-m-d H:i:s.u'
+     * @param float $timestamp
+     * @return string
+     */
+    protected function getTime($timestamp)
+    {
+        list ($timestamp, $usec) = explode('.', (string)$timestamp);
```

Comment:
What's `usec`?

Reply:
I think it should be named millisecond.

Reply:
`usec` is `μsec`. For example, function `gettimeofday` returns array with keys "sec" and "usec".